voice/src/main/java/discord4j/voice/DefaultVoiceGatewayClient.java

```diff
@@ -260,10 +265,21 @@ private Mono<Void> connect(VoiceServerOptions vso, String sessionId,
                                                                 innerCleanup.add(connection);
                                                                 String hostName = address.getHostName();
                                                                 int port = address.getPort();
+
+                                                                this.encryptionMode = EncryptionMode.getBestMode();
+                                                                if (this.encryptionMode == null) {
+                                                                    log.error("No encryption mode available");
```

Comment:
this just not make show two messages of error?
one this log and latert the voiceConnectionSink?